package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sanctuary-def",
+  "version": "0.0.0",
+  "description": "Define strict, curried functions",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plaid/sanctuary-def.git"
+  },
+  "dependencies": {
+    "ramda": "0.18.x"
```

Comment:
I've removed all uses of Ramda but one:

```javascript
var commonTypes = R.intersection(types, actualTypes);
```

This revealed something unfortunate: the result is not guaranteed to be a subset of *either* argument, due to the quirky equality semantics of Type values. Worse, the code *relies* on this behaviour. I'm not happy with this situation. Although I'm very much looking forward to using sanctuary-def in Sanctuary and in other projects, and I'm happy with its current API, I'm reluctant to merge an initial pull request containing code which is unclear to me even after spending dozens of hours with it. Thanksgiving will provide me an opportunity to rethink my approach. I'd like to provide the same API with dramatically simpler internals, to provide a solid foundation for future enhancements and to allow others to understand the code and contribute fixes.